imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -110,7 +110,7 @@ esa_step_coord:
 shcoarse:
   <<: *default_uint32
   CATDESC: Mission Elapsed Time
-  DEPEND_0: epoch
+  DEPEND_1: shcoarse
```

Comment:
@sdhoyt, I was looking at this and confused. I thought this was the new MET because the CATDESC is Mission Elapsed Time.

How can shcoarse be dependent on itself?

Reply:
My plan was to make shcoarse a coordinate that the CCSDS header info would depend on, but I want to check with Lo to see if they want the headers and I was going to circle back later to finish that anyway. From what I've seen it looks like coordinates are supposed to depend on themselves. It's possible I'm doing that wrong though.

Reply:
And yeah, I need to update the descriptions to make things more clear

Reply:
> From what I've seen it looks like coordinates are supposed to depend on themselves. It's possible I'm doing that wrong though.

👍 It just looked odd to me, if that is what it is supposed to be then no problem at all!